tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -90,6 +118,63 @@
 ]
 
 INVALID: List[Container] = [
+    Container(
+        name="retf_stack_validation_1",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=2),
+            Section.Code(
+                code=Op.PUSH0 + Op.RETF,
+                code_outputs=2,
+                max_stack_height=1,
+            ),
+        ],
+        expected_bytecode="ef000101000802000200040002040000000080000200020001e30001005fe4",
+        validity_error=EOFException.STACK_UNDERFLOW,
+    ),
+    Container(
+        name="retf_variable_stack_0",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=5),
+            Section.Code(
+                code=Op.PUSH0 + Op.PUSH1[0] + Op.RJUMPI[2] + Op.PUSH0 * 2 + Op.RETF,
```

Comment:
I think both branches underflow in this case, it would be nice to see:
- ~~Case where it only underflows in the jump branch~~ (Edit: covered in case below)
- Case where it only underflows in the no-jump branch (no-jump pops from the stack for example).

Reply:
I added this case underflowing in the no-jump branch: https://github.com/ethereum/execution-spec-tests/blob/3f30efd14aaf79e252b23168b3b2317031c3a9a5/tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py#L155-L166